### DISCUSSION TITLE
好奇一下为什么一个网站需要1.2GB的空间?

### DISCUSSION THREAD

**Discussion by chewhg78** [ant-design/ant-design-pro#10180] [General]
我是新人，好奇一下为什么一个网站会需要1.2GB的硬盘空间?

![big_size](https://user-images.githubusercontent.com/41529359/187416353-2a0ff248-2f89-4017-b8a7-086d5b16e31c.png)

**Comment by neveler** [comment]


**Reply by chewhg78** [reply]
就是好奇构建前为什么会需要那么大的空间
